proto/sentry_protos/snuba/v1/trace_item_attribute.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+message AttributeKey {
+  enum Type { //this mostly reflects what types are able to be ingested, see eap_spans consumer for ingest details
+    TYPE_UNSPECIFIED = 0; //called "none" sometimes
```

Comment:
isn't "none" in the context of units? would we ever do anything but error if someone asked for an untyped attribute?

Reply:
This is a carryover from V0, I'll remove it